source/deployment/advanced-permissions.rst

```diff
@@ -0,0 +1,191 @@
+
+.. note::
+*Available in a future release of Enterprise Edition E20*
+
+Allows Admins to restrict permissions within specific channels. Permissions under consideration for this phase include:
+
+- **Read-only Channels:** The ability for Admins to turn off posting in specified channels.
+- **Restrict Channel Mentions:** Turn off the ability for users to post channel wide mentions (@-all/channel/here) in specified channels.
+- **Channel member management:** Restricting adding and removing channel members to Admins only in specified channels.
+
+Supplementary Roles (E20)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. note::
+*Available in a future release of Enterprise Edition E20*
+
+Allows Admins to grant additional permissions to specific users or to a group of users based on AD/LDAP group membership. Permissions can be granted within the scope of channels, teams or system level.
+
+Recipes
+--------
+This section provides some examples of common permissions use cases and how to accomplish them using the Advanced Permissions System Console interface.
+
+Team Management
+~~~~~~~~~~~~~~~~
+
+**Restict adding members to only Admins, in specific teams**
+Example: In Team A, only allow Team and System Admins to add new team members. As the default for all other teams, allow all users to add and invite new members.
+
+1. In **System Console** > **Advanced Permissions**, edit the **System Scheme**
+2. In the **All Members** panel, check the box for **Add Team Members**. This sets the system default for all teams.
+3. In **System Console** > **Advanced Permissions**, create a new **Team Override Scheme**
+  i. Name and describe the scheme. For example, ``Authorized Personel Only`` with description ``Restrict adding team members to Team and System Admins.``
+  ii. Add Team B to the **Select teams to override permissions** list.
+  iii. In the **All Members** panel, uncheck the box for **Add Team Members**.
+  iv. In the **Team Admin** and **Channel Admin** panels, check the box for **Add Team Members**. 
+
+
+Public and Private Channel Management
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+**Restrict who can rename channels and edit channel header and purposes**
+Example: As the default for the entire system, restrict renaming channels and editing headers and purposes to Admins only.
+
+1. In **System Console** > **Advanced Permissions**, edit the **System Scheme**
+2. In the **All Members** panel, uncheck the box for **Manage Channel Settings**
+3. In the **Team Admin** and **Channel Admin** panels, check the box for **Manage Channel Settings**
+
+.. note::
+
+  Permissions for channel renaming, editing header and editing purpose are currently grouped in a single permission. These will be split into separate permissions in a future release.
+
+**Restrict who can create channels, in specific teams**
+
+Example: In Team C, restrict public channel creation to Admins. As the default for all other teams, allow everyone to create public channels.
```

Comment:
Instead of "Team C" or "Team B", another option might be to use more specific examples like "I want to restrict public channel creation in my InfoSec team to Admins only. Here's how I can do it".

Makes it easier to relate to / apply to your own team, and less abstract.

^another nice-to-have